src/main/java/de/tum/in/www1/artemis/web/websocket/repository/RepositoryWebsocketService.java

```diff
@@ -0,0 +1,119 @@
+package de.tum.in.www1.artemis.web.websocket.repository;
+
+import afu.org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
I guess you mean import javax.annotation.Nullable;
```suggestion
import javax.annotation.Nullable;
```

Reply:
You are right, thanks!